src/GeositeFramework/js/Export.js

```diff
@@ -34,71 +34,162 @@ require(['use!Geosite'],
+                    $('<link>', {
+                        rel: 'stylesheet',
+                        href: pageCssLink(pageOrientation),
+                        'class': '.print-orientation-css',
+                    }).appendTo('head');
+                    _.delay(orientDeferred.resolve, 500);
+
+                    $.when(orientDeferred).then(function () {
+                        context.map.width = parseFloat(exportMap.css("width"));
+                        context.map.height = parseFloat(exportMap.css("height"));
+                        context.map.resize(true);
+                        context.map.centerAt(context.mapDimensions.extent.getCenter());
+                        _.delay(resizeDeferred.resolve, 500);
+
+                        // the legend items are affected by the map resize, so
+                        // they are manipulated just before printing
+                        $.when(resizeDeferred).then(function() {
+                            // expand legend container if minimized
+                            if (context.legend.hasClass("minimized")) {
+                                $(".legend-close").click();
+                                postPrintAction = function() {
+                                    $(".legend-close").click();
+                                };
+                            }
 
                             if ($("[name='export-include-legend']").is(":checked")) {
                                 // show & expand all legend items
-                                $("#legend-container-0").css({ visibility: "visible" });
-                                _.each(
-                                    $(".item.expand>.expand-legend"),
-                                    function (el) {
-                                        el.click();
-                                    });
+                                context.legend.css({ visibility: "visible" });
+                                $(".item.expand>.expand-legend").click();
                                 $(".item.extra.collapse").hide();
+
                             } else {
                                 // if the style rule is changed via jquery, that state seems to
                                 // "stick", regardless of what's in the stylesheet
-                                $("#legend-container-0").css({ visibility: "hidden" });
+                                context.legend.css({ visibility: "hidden" });
                             }
+                            // wrap all legend items in a div to style separately from header
+                            $(".legend-layer").each(
+                                function(el) {
+                                    $(this)
+                                        .children(".item")
+                                        .wrapAll("<div class='print-legend-coll'></div>");
+                            });
```

Comment:
I guess I don't really see the point to wrapping these. You should be able to override the legend item styles in the print stylesheet. It seems a little redundant to wrap each legend item div with another div.